src/utils/integration/airtable.js

```diff
@@ -0,0 +1,65 @@
+const { getApiData } = require('./readApi.js');
+const { stringify } = require('csv-stringify/sync');
+
+const cols = [
+  'Id',
+  'Nome do Abrigo',
+  'Suprimentos necessários',
+  'Contato',
+  'Capacidade',
+  'Número de abrigados atualmente',
+  'Endereço',
+  'pix',
+  'Verificado',
+  'Aceita animais',
+  'Criado em',
+  'Atualizado em',
+];
+
+const TABLE_URI = '';
```

Comment:
```suggestion
const TABLE_URI = '';
```
Move this to env var for better scalability 😀
Same case with for token.